EventsExpress/ClientApp/src/components/event/map/location-map.js

```diff
@@ -16,10 +17,28 @@ const mapStyles = {
 class LocationMap extends Component {
     constructor(props) {
         super(props);
+        let value = null;
+        let start = [50.4547, 30.5238];//start position by default
```

Comment:
This code duplicates the logic from `getStartPosition` method. Please use it here.